src/jdk.jfr/share/classes/jdk/jfr/internal/instrument/JDKEvents.java

```diff
@@ -59,11 +56,8 @@
 public final class JDKEvents {
 
     private static final Class<?>[] eventClasses = {
-        FileForceEvent.class,
         FileReadEvent.class,
         FileWriteEvent.class,
-        SocketReadEvent.class,
-        SocketWriteEvent.class,
```

Comment:
Yes, and I think it might be the cause for https://bugs.openjdk.org/browse/JDK-8329330 I will send out a PR to remove those separately and so the fix can be backported.

Reply:
That's correct, it is an unrelated cleanup (other than it seems to cause tests to fail now).  @egahlin, do you want me to remove those from this PR?

Reply:
Yes, it would be good if you could remove those changes, so it can be handled separately.